actions/send-sves.vbs

```diff
@@ -115,21 +113,22 @@ EMReadScreen SSN2, 2, 7, 46
 EMReadScreen SSN3, 4, 7, 49
 EMReadScreen PMI, 8, 4, 46
 
-If SSN_radiobutton = 1 then
-    call navigate_to_MAXIS_screen("infc", "sves")
-    'checking for IRS non-disclosure agreement.
-    EMReadScreen agreement_check, 9, 2, 24
-    IF agreement_check = "Automated" THEN script_end_procedure("To view INFC data you will need to review the agreement. Please navigate to INFC and then into one of the systems and review the agreement.")
+If SVES_actions = "SSN" then
+    CALL navigate_to_MAXIS_screen("INFC" , "SVES")
+	'checking for NON-DISCLOSURE AGREEMENT REQUIRED FOR ACCESS TO IEVS FUNCTIONS'
```

Comment:
Will this code work just navigating to INFC/SVES, or do you need to go further?

Reply:
it does seem to work Casey had tested just going into SVES no further and got the agreement